app/src/main/res/values-uk/strings.xml

```diff
@@ -19,59 +19,70 @@
 
     <string name="search_hint">Пошук нотаток</string>
-    <string name="share_string">Поділитися з...</string>
-    <string name="share">Поділитися</string>
+    <string name="share_string">Надіслати до...</string>
+    <string name="share">Надіслати</string>
     <string name="search">Пошук</string>
-    <string name="preview">Предперегляд</string>
+    <string name="preview">Попередній перегляд</string>
     <string name="app_settings">Налаштування додатку</string>
     <string name="editor_settings">Налаштування редактора</string>
     <string name="folders">Директорії</string>
     <string name="files">Файли</string>
 
     <!-- Preferences -->
-    <string name="pref_title__remember_directory">Пам’ятати останню відкриту директорію</string>
+    <string name="pref_title__remember_directory">Запам\'ятовувати останню відкриту директорію</string>
     <string name="pref_title__font_choice">Шрифт редактора нотаток</string>
     <string name="pref_title__theme">Тема</string>
-    <string name="pref_title__preview_first">Спочатку показувати предперегляд при відкритті нотатки</string>
-    <string name="about">Про додаток Markor</string>
-    <string name="share_text">Просто текст</string>
+    <string name="pref_title__preview_first">Показувати попередній перегляд при відкритті нотатки</string>
+    <string name="about">Про Markor</string>
+    <string name="share_text">Звичайний текст</string>
+    <string name="share_file">Файл</string>
     <string name="share_image">Зображення</string>
+    <string name="share_pdf">PDF</string>
     <string name="pref_title__font_size">Розмір шрифту</string>
-    <string name="pref_title__show_markdown_shortcuts">Показувати скорочення markdown</string>
+    <string name="pref_title__show_markdown_shortcuts">Показувати панель скорочень markdown</string>
     <string name="pref_title__highlighting_activated">Підсвітка синтаксису markdown</string>
     <string name="pref_title__highlighting_delay">Затримка оновлення підсвітки синтаксису в мілісекундах</string>
     <string name="pref_title__highlighting_delay_summary">Великі значення менше витрачають батарею</string>
     <string name="pref_title__smart_shortcuts">Розумні дужки</string>
     <string name="pref_title__smart_shortcuts_summary">{}, [] и () будуть оточувати виділений текст</string>
-    <string name="pref_title__render_rtl">Працювати з мовами \"справа-наліво"</string>
-    <string name="error_cannot_create_save_dir">Помилка: неможливо зберегти директорію</string>
+    <string name="pref_title__render_rtl">Відображати для писемності справа-наліво</string>
+    <string name="error_cannot_create_save_dir">Помилка: неможливо створити директорію</string>
 
     <!-- Font Size -->
 
     <!-- Hard coded string fix -->
     <string name="last_modified">Дата зміни: %s</string>
-    <string name="number_of_files">Кількість файлів: %d</string>
+    <string name="number_of_files">Число файлів: %d</string>
```

Comment:
Why do you think new one is better? Old one seems to me more native.

Reply:
Agree.